.github/workflows/auto-format.yml

```diff
@@ -0,0 +1,28 @@
+name: Auto-Formatting
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up JDK 11
+      uses: actions/setup-java@v1
+      with:
+        java-version: 11
+    - name: Run Spotless
+      run: ./gradlew spotlessApply
```

Comment:
We may want `--parallel` here in case it speeds up this action

Reply:
Good idea. AFAIK the container is allowed to use 2 cores.